service/dynamodb/expression/operand.go

```diff
@@ -11,122 +9,80 @@ import (
 )
 
 var (
-	// ErrEmptyPath is an error that is returned if the path specified is or has
+	// ErrEmptyName is an error that is returned if the name specified is or has
 	// an empty string. (i.e. "", "foo..bar")
-	ErrEmptyPath = awserr.New("EmptyPath", "BuildOperand error: path is an empty string", nil)
+	ErrEmptyName = awserr.New("EmptyName", "BuildOperand error: name is an empty string", nil)
 
-	// ErrInvalidPathIndex is an error that is returned if the index used in the
-	// path either does not have a path associated with it or is an empty string.
+	// ErrInvalidNameIndex is an error that is returned if the index used in the
+	// name either does not have an item attribute associated with it or is an
+	// empty string.
 	// (i.e. "[2]", "foo[]")
-	ErrInvalidPathIndex = awserr.New("InvalidPathIndex", "BuildOperand error: invalid path index", nil)
+	ErrInvalidNameIndex = awserr.New("InvalidNameIndex", "BuildOperand error: invalid name index", nil)
 )
 
 // ValueBuilder represents a value operand and will implement the OperandBuilder
 // interface. It will have various methods corresponding to the operations
 // supported by DynamoDB operations. (i.e. AND, BETWEEN, EQUALS) The underlying
 // undefined type member variable will be converted into dynamodb.AttributeValue
-// by dynamodbattribute.Marshal(in) when Expressions are created.
+// by dynamodbattribute.Marshal(in) when Builder are created.
 type ValueBuilder struct {
 	value interface{}
 }
 
-// PathBuilder represents a path to either a top level item attribute or a
-// nested attribute. It will implement the OperandBuilder interface. It will
+// NameBuilder represents a name of a top level item attribute or a nested
+// attribute. It will implement the OperandBuilder interface. It will
 // have various methods corresponding to the operations supported by DynamoDB
 // operations. (i.e. AND, BETWEEN, EQUALS)
```

Comment:
Should state that a `Name` is a `DynamoDB Operand` to create that clear relationship.